src/Entity/Value.php

```diff
@@ -60,6 +60,15 @@
      */
     protected $attribute;
 
+    /**
+     * @Serializer\VirtualProperty
+     * @Serializer\SerializedName("attributeTranslations")
+     * @Serializer\Groups("translations_value_view")
```

Comment:
translations_value_view -> eav_value_view_title_translations
eav_value - entity code
view - means extra groups
title_translations - return transltions of title